perception/sub8_perception/CMakeLists.txt

```diff
@@ -133,6 +133,10 @@ install(DIRECTORY include/${PROJECT_NAME}/
 #     "src/sub8_perception/*.cc"
 # )
 
+file(GLOB START_GATE_SOURCES
+      "src/sub8_perception/start_gate.cc"
```

Comment:
_Throws to the ground and kicks_ Me and @ironmig at the very least agreed that the good reason to ignore this part of it is because .h is for c files while .hpp is for c++.